docs/ops.html

```diff
@@ -586,7 +582,7 @@
   socket.request.max.bytes=104857600
   socket.receive.buffer.bytes=1048576
   socket.send.buffer.bytes=1048576
-  queued.max.requests=16
+  queued.max.requests=500
```

Comment:
@amethystic : One thing that we need to be a bit careful is that ReplicaFetchThread issues fetch requests very aggressively (with min.byte=1). Having more fetch threads increases the number of fetch requests from the follower, which tends to be a significant portion of total requests on a broker. There is a bit of overhead per request. Too many requests can exhaust the available request handling threads. So, if the follower can keep up with just 1 num.replica.fetchers, I feel it's better not to increase it.